rollbar-web/src/main/java/com/rollbar/web/provider/RequestProvider.java

```diff
@@ -0,0 +1,103 @@
+package com.rollbar.web.provider;
+
+import com.rollbar.api.payload.data.Request;
+import com.rollbar.notifier.provider.Provider;
+import com.rollbar.web.listener.RollbarRequestListener;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * {@link Request} provider.
+ */
+public class RequestProvider implements Provider<Request> {
+
+  /**
+   * Constructor.
+   */
+  public RequestProvider() {
+  }
+
+  @Override
+  public Request provide() {
+    HttpServletRequest req = RollbarRequestListener.getServletRequest();
+
+    if(req != null) {
+      Request request = new Request.Builder()
+          .url(url(req))
+          .method(method(req))
+          .headers(headers(req))
+          .get(getParams(req))
+          .querystring(queryString(req))
```

Comment:
Can we rename this to be `queryString(...)`?

Reply:
Sure, I'll rename it.